SwagLyricsBot/dev_commands.py

```diff
@@ -41,8 +39,9 @@ async def view_stats(self, ctx):
         embed.add_field(name="Bot Uptime",
                         value=f'{timedelta(seconds=int(time.time() - process.create_time()))}')
         embed.set_footer(
-            text=f"Made by {app_info.owner}",
-            icon_url=app_info.owner.avatar_url_as(size=128))
+            text=f"Made by {app_info.owner} • clash#1337",
```

Comment:
```suggestion
            text=f"Made by {app_info.owner} • self.bot.get_user(512708394994368548)",
```

In case you ever change your tag/username.

Reply:
you mean when nitro ends :joy:

Reply:
done